compose/local/postgres/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM postgres:12.3
```

Comment:
Could you explain a bit more about the maintenance and backup scripts that were added here? And is there anything notable that needs to happen in order to start using this image?

For example, on the ongoing test run in test.katagodistributed.org, is it still sufficient to simply build, migrate, and restart the server - will the new postgres image still connect fine to the old database volume and keep working without any special changes? Thanks!

Reply:
Oh wait, this is all local. Is there a reason for doing all this work for "local" but not add the same maintenance and backup scripts for "production"?

Reply:
Ooooh. We did have this config in production. Okay. Is the reason for copying it to local simply to make it more consistent, or is there a direct use case in local right now that you needed to fix a problem when changing the image?

Reply:
We were using the production postgres inside local.yml.
Which is weird, for me.

Reply:
I didn't change anything but it is normal to have different query in postgres for dev and prod.

Reply:
Eg: http://www.louisemeta.com/blog/developers-and-logs/